src/main/java/com/todoist/markup/EmojiParser.java

```diff
@@ -1,21 +1,18 @@
k-out_tongue_and_winking_eye:\": \"\uD83D\uDE1C\",\":black_right-pointing_double_triangle_with_vertical_bar:\": \"⏭\",\":squared_cjk_unified_ideograph-7981:\": \"\uD83C\uDE32\",\":vhs:\": \"\uD83D\uDCFC\",\":wavy_dash:\": \"〰\",\":bed:\": \"\uD83D\uDECF\",\":flag_for_Timor-Leste:\": \"\uD83C\uDDF9 \uD83C\uDDF1\",\":information_desk_person:\": \"\uD83D\uDC81\",\":white_heavy_check_mark:\": \"✅\",\":flag_for_Vanuatu:\": \"\uD83C\uDDFB \uD83C\uDDFA\",\":fearful:\": \"\uD83D\uDE28\",\":currency_exchange:\": \"\uD83D\uDCB1\",\":arrow_right:\": \"➡\",\":anger_symbol:\": \"\uD83D\uDCA2\",\":triumph:\": \"\uD83D\uDE24\",\":construction_worker:\": \"\uD83D\uDC77\",\":tophat:\": \"\uD83C\uDFA9\",\":cloud_with_tornado:\": \"\uD83C\uDF2A\",\":water_closet:\": \"\uD83D\uDEBE\",\":heavy_minus_sign:\": \"➖\",\":baby:\": \"\uD83D\uDC76\",\":wave:\": \"\uD83D\uDC4B\",\":black_square_button:\": \"\uD83D\uDD32\",\":turtle:\": \"\uD83D\uDC22\",\":hear-no-evil_monkey:\": \"\uD83D\uDE49\",\":basketball_and_hoop:\": \"\uD83C\uDFC0\",\":m:\": \"Ⓜ\",\":stopwatch:\": \"⏱\",\":closed_lock_with_key:\": \"\uD83D\uDD10\",\":baggage_claim:\": \"\uD83D\uDEC4\",\":clock7:\": \"\uD83D\uDD56\",\":cheering_megaphone:\": \"\uD83D\uDCE3\",\":baby_angel:\": \"\uD83D\uDC7C\",\":star_of_david:\": \"✡\",\":shopping_bags:\": \"\uD83D\uDECD\",\":black_left-pointing_double_triangle:\": \"⏪\",\":flag_for_Honduras:\": \"\uD83C\uDDED \uD83C\uDDF3\",\":disappointed:\": \"\uD83D\uDE1E\",\":chains:\": \"⛓\",\":lipstick:\": \"\uD83D\uDC84\",\":card_file_box:\": \"\uD83D\uDDC3\",\":meat_on_bone:\": \"\uD83C\uDF56\",\":arrow_lower_right:\": \"↘\",\":volcano:\": \"\uD83C\uDF0B\",\":japanese_post_office:\": \"\uD83C\uDFE3\",\":video_game:\": \"\uD83C\uDFAE\",\":bug:\": \"\uD83D\uDC1B\",\":art:\": \"\uD83C\uDFA8\",\":high_voltage_sign:\": \"⚡\",\":full_moon:\": \"\uD83C\uDF15\",\":peach:\": \"\uD83C\uDF51\",\":horizontal_traffic_light:\": \"\uD83D\uDEA5\",\":u55b6:\": \"\uD83C\uDE3A\",\":eyeglasses:\": \"\uD83D\uDC53\",\":up-pointing_red_triangle:\": \"\uD83D\uDD3A\",\":frame_with_picture:\": \"\uD83D\uDDBC\",\":spades:\": \"♠\",\":lips:\": \"\uD83D\uDC44\",\":eight_pointed_black_star:\": \"✴\",\":carousel_horse:\": \"\uD83C\uDFA0\",\":grey_question:\": \"❔\",\":grinning_cat_face_with_smiling_eyes:\": \"\uD83D\uDE38\",\":leaf_fluttering_in_wind:\": \"\uD83C\uDF43\",\":flag_for_Norfolk_Island:\": \"\uD83C\uDDF3 \uD83C\uDDEB\",\":clock130:\": \"\uD83D\uDD5C\",\":earth_americas:\": \"\uD83C\uDF0E\",\":mute:\": \"\uD83D\uDD07\",\":rosette:\": \"\uD83C\uDFF5\",\":ship:\": \"\uD83D\uDEA2\",\":clockwise_rightwards_and_leftwards_open_circle_arrows_with_circled_one_overlay:\": \"\uD83D\uDD02\",\":flag_for_Thailand:\": \"\uD83C\uDDF9 \uD83C\uDDED\",\":candle:\": \"\uD83D\uDD6F\",\":shamrock:\": \"☘\",\":umbrella_with_rain_drops:\": \"☔\",\":satellite_antenna:\": \"\uD83D\uDCE1\",\":derelict_house_building:\": \"\uD83C\uDFDA\",\":flag_for_Christmas_Island:\": \"\uD83C\uDDE8 \uD83C\uDDFD\",\":repeat:\": \"\uD83D\u
```

Comment:
Could we put this line in separate class? It will make reading this class easier.